client/src/component/Search/ListEntry.jsx

```diff
@@ -5,16 +5,16 @@ const ListEntry = ({toy}) => {
   return (
     <div className={styles.itemContainer}>
       <div>
-        <img src={toy.img} width="100" height="100"/>
+        <img src={toy.photos[0]} width="100" height="100"/>
       </div>
       <div className={styles.search}>
       <p>{toy.name}</p>
-      <p>{toy.price}</p>
-      <p>By {toy.seller}</p>
+      <p>{toy.price.original}</p>
+      {/* <p>By {toy.seller}</p> */}
       </div>
       <div >
       <button>Add to cart</button>
-      { toy.stock > 2 ? <p> {toy.stock} IN STOCK</p> : <p>LOW IN STOCK</p> }
+      { toy.stock > 2 ? <p> {toy.quantity} IN STOCK</p> : <p>LOW IN STOCK</p> }
```

Comment:
do we have something for a product being out of stock?

Reply:
yes it's comming in my next pr

Reply:
this one for search only